Source/WebCore/page/InteractionRegion.cpp

```diff
@@ -307,7 +367,8 @@ std::optional<InteractionRegion> interactionRegionForRenderedRegion(RenderObject
     bool isPhoto = false;
 
     float minimumPhotoArea = 200 / scale * 200 / scale;
-    if (bounds.area() > minimumPhotoArea) {
+    bool bigEnoughForPhoto = bounds.area() > minimumPhotoArea;
```

Comment:
Not sure I love this name. is it really about "photo" or just "being primary content" of some sort?